imap_processing/glows/tests/test_glows_decom.py

```diff
@@ -11,8 +12,8 @@
 def decom_test_data():
     """Read test data from file"""
     current_directory = Path(__file__).parent
-    packet_path = f"{current_directory}/glows_test_packet_20230920_v00.pkts"
-    data_packet_list = decom_glows.decom_packets(packet_path)
+    packet_path = current_directory / "glows_test_packet_20230920_v00.pkts"
+    data_packet_list = decom_glows.decom_packets(str(packet_path))
```

Comment:
Good catch. I shortened the packet file, since I didn't need something that large for unit testing. Therefore, the test file used here only contains part of the packet file which was provided.

Reply:
I reworked the test packet file so it is now small enough to put in github and run tests quickly, but also so it includes all the expected direct event and histogram packets. So, those numbers are now updated back to 505 and 1088.